charts/sn-platform-vault/templates/vault.yaml

```diff
@@ -81,11 +81,11 @@ spec:
       {{- include "vault.storage.class" . | nindent 6 }}
 {{- else }}
   volumes:
-  - name: "{{ template "pulsar.fullname" . }}-{{ .Values.vault.component }}-{{ .Values.vault.volume.name }}"
+  - name: "{{ template "pulsar.fullname" . }}{{ template "vault.component" . }}-{{ .Values.vault.volume.name }}"
     emptyDir: {}
   {{- end }}
   volumeMounts:
-    - name: "{{ template "pulsar.fullname" . }}-{{ .Values.vault.component }}-{{ .Values.vault.volume.name }}"
+    - name: "{{ template "pulsar.fullname" . }}{{ template "vault.component" . }}-{{ .Values.vault.volume.name }}"
```

Comment:
```suggestion
    - name: "{{ template "pulsar.fullname" . }}{{ template "vault.component" . }}-{{ .Values.vault.volumeMount.name }}"
```

Reply:
This name needs to match the name of `volumeClaimTemplates` or `volume` .

Reply:
oh, sorry, my bad. It seems like we don't need this [volumeMounts](https://github.com/streamnative/charts/blob/ef06c3658233caacf4a821eef36030f952991a7e/charts/sn-platform-vault/values.yaml#L111) anymore.